packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -93,15 +94,26 @@ import InMeetingActions from './in-meeting-actions';
  * @property {Boolean} sendShare
  */
 
+/**
+  * Media Ready Event
+  * Emitted when ever there is a meeting state change
+  * @event meeting:stateChange
+  * @instance
+  * @type {Object}
+  * @property {string} currentState previous state of the meeting
+  * @property {String} previousState previous state of the meeting
+  * @memberof Meeting
+  */
+
 /**
   * Media Ready Event
   * Emitted when a stream is ready to be rendered
   * @event media:ready
   * @instance
   * @type {Object}
-  * @property {MediaStream} stream the media stream
+  * @property {string} stream the media stream
```

Comment:
The stream is a string? Also, please uppercase `String`

Reply:
my bad it should be mediaStream